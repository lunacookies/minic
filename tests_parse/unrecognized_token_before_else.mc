func main {
	if x
		set x = 1 $
	else {}
}
