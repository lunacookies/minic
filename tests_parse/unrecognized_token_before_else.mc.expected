func main {
	if x <missing>
	<missing>
	<missing>
	{}
}
tests_parse:30..31: error: invalid token “$”
tests_parse:20..29: error: expected block but found a single statement
tests_parse:30..31: error: expected statement but found unrecognized token
tests_parse:33..37: error: unmatched “else”
